vault.cabal

```diff
@@ -45,6 +45,8 @@ Library
                         hashable >= 1.1.2.5 && < 1.3
 
     extensions:         CPP
+    if flag(UseGHC) && impl(ghc >= 7.8)
+      extensions:       RoleAnnotations
```

Comment:
Could you move the `RoleAnnotations` language extension from the cabal file to the source file by using a `LANGUAGE` pragma?